### PR TITLE
fix release notes

### PR DESCRIPTION
**Pull request by nslater**
per discussion

these changes have been reviewed visually by @seut

### CODE REVIEW

**Comment by nslater** [comment]
@dobe just discussed this with @seut. I'm gonna make changes to fix things up in line with the call you both just had

**Comment by matriv** [comment]
@nslater Please don't forget to adjust the processing of the changelog files for the debian/ubuntu packages here: https://github.com/crate/distribute/tree/mt/fix-parsing-changes

**Comment by nslater** [comment]
@matriv good catch thank you

File: blackbox/docs/release_notes/1.1.1.txt [hunk 1]
```diff
@@ -6,6 +6,14 @@ Version 1.1.1
 
 Released on 2017/03/27.
 
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.0.0` or higher before you upgrade to 1.1.1.
```
Comment (seut) [inline blackbox/docs/release_notes/1.1.1.txt]: pls change to 0.57.0

Comment (nslater) [inline blackbox/docs/release_notes/1.1.1.txt]: @seut fixed this for every version

**Comment by nslater** [comment]
note: @matriv's comment was addressed in crate/distribute#182

**Comment by nslater** [comment]
this pr currently has two commits on it. first represents changes discussed with @dobe and @seut. the second represents changes re upgrading requested by @mfussenegger as part of a separate impediment. will squash once things have been acked

**Pull request merged**